github_api/repos.py

```diff
@@ -29,6 +29,4 @@ def set_desc(api, urn, desc):
 def get_creation_date(api, urn):
     """ returns the creation date of the repo """
     data = api("get", get_path(urn))
-    # this is the field for watchers.  do not be tricked by "watchers_count"
-    # which always matches "stargazers_count"
-    return datetime.strptime(data["created_at"], "%Y-%m-%dT%H:%M:%SZ").date()
```

Comment:
Also, how in the world did the PR get through with this comment? Did no one read the changes?